File: todo-api/ToDoApis/Models/ToDoModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace ToDoApis.Models
{
    public class ToDoModel
    {
        public long Id { get; set; }
        [Required]
        public string Title { get; set; }
        [Required]
        public string Description { get; set; }
        public long UserId { get; set; }
        public string Status { get; set; }
    }
}

File: todo-ui/src/models/ToDoModel.js

  const TaskState = Object.freeze({
    TODO : "ToDo",
    DONE: "Done"
});     

  var ToDoModel={
      id:0,
      title:"",
      description:"",
      userId:"",
      status:TaskState.TODO
      }

 

 export { ToDoModel,TaskState};      
